esp/GBPlay/main/GBPlay.c

```diff
@@ -35,8 +36,10 @@ void init_console()
 
 void start_tasks()
 {
-    task_status_indicator_start();
-    task_connection_manager_start();
+    task_network_manager_start(0 /* core */, 2 /* priority */);
```

Comment:
I changed how priorities are set so we can easily configure it all in one place and see it at a glance.